### PR TITLE
docs: `export_project` tutorial

### PR DESCRIPTION
**Pull request by Jonas1312**
TODO:
- [x] add hidden cells to create the project with the assets, so that the notebook runs end-to-end
- [x] add project delete at the end of the notebook (hidden cell)
- [x] make sure tqdm doesn't show in the doc
- [x] wait for #882 to be merged, rebase this branch on master, and convert the notebook to markdown
- [x] make sure the tutorial is rendered properly in mkdocs
- [x] add link to google colab

@xczuba I made a lot of modifications in this notebook. It would be nice if you could check if it's still ok with what we want to show in the documentation

### CODE REVIEW

File: recipes/tutorials/export_a_kili_project.ipynb [hunk 1]
```diff
@@ -0,0 +1,888 @@
+{
```
Comment (PierreLeveau) [inline recipes/tutorials/export_a_kili_project.ipynb:1]: <p>The output shows an error: <code>Project needs at least one OBJECT_DETECTION task.</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/885/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Comment (Jonas1312) [reply recipes/tutorials/export_a_kili_project.ipynb:1]: I just ran it twice, once in jupyter lab, once in pytest, it ran without any issues.

did you run the notebook on master?

**Review by PierreLeveau — APPROVED** [review]

File: recipes/tutorials/export_a_kili_project.ipynb (hunk 1 above)
Comment (xczuba) [inline recipes/tutorials/export_a_kili_project.ipynb:1]: <p>Set up</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/885/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (xczuba) [inline recipes/tutorials/export_a_kili_project.ipynb:1]: <p>One can -&gt; You can</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/885/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Comment (xczuba) [inline recipes/tutorials/export_a_kili_project.ipynb:1]: <p>on project -&gt; on a project</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/885/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

**Comment by Jonas1312** [comment]
we can merge

**Pull request merged**